src/use-swr-infinite.ts

```diff
@@ -100,7 +100,7 @@ function useSWRInfinite<Data = any, Error = any>(
 
   // actual swr of all pages
   const swr = useSWR<Data[], Error>(
-    firstPageKey ? ['inf', firstPageKey] : null,
+    firstPageKey ? ['$inf', firstPageKey] : null,
```

Comment:
```suggestion
    firstPageKey ? 'inf@' + firstPageKey : null,
```
since firstPageKey is already a string here, how about giving it a `inf@` prefix? what's your thought here? @shuding

Reply:
Sounds great to me! @huozhi

Reply:
Why not adding `$` btw?